single-pool/cli/Cargo.toml

```diff
@@ -28,7 +28,7 @@ solana-sdk = "=1.16.3"
 solana-transaction-status = "=1.16.3"
 solana-vote-program = "=1.16.3"
 spl-token = { version = "4.0", path="../../token/program", features = [ "no-entrypoint" ] }
-spl-token-client = { version = "0.5", path="../../token/client" }
+spl-token-client = { version = "0.6", path="../../token/client" }
```

Comment:
Same with this CLI, doesn't bump when imports bump?

Reply:
I'll let @2501babe bump and release when it makes sense. I don't think the single pool CLI needs a new release based on this change.